static/css/_forms.scss

```diff
@@ -104,6 +100,7 @@ input[type="submit"] { display: block; }
 
 //  Writing page
 form.new-post {
+  max-width: 60em;
```

Comment:
This will enforce the max-width for all element inside so we don't have to define the max-width for every element that can be used in a form element